api/middleware/contentLengthLimiter.go

```diff
@@ -0,0 +1,84 @@
+package middleware
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/multiversx/mx-chain-go/api/shared"
+
+	"github.com/multiversx/mx-multi-factor-auth-go-service/config"
+)
+
+const (
+	unknownContentLengthSize = -1
+	minSizeBytes             = 10
+)
+
+type contentLengthLimiter struct {
+	maxContentLengths map[string]uint64
+}
+
+// NewContentLengthLimiter will abort all requests that have Content-Length size bigger
+// than the one specified in config.
+func NewContentLengthLimiter(apiPackages map[string]config.APIPackageConfig) (*contentLengthLimiter, error) {
+	maxContentLengths := map[string]uint64{}
+	for group, groupCfg := range apiPackages {
+		groupPath := fmt.Sprintf("/%s", group)
+
+		for _, r := range groupCfg.Routes {
+			if r.MaxContentLength < minSizeBytes {
```

Comment:
here also the check for get
```go
		if c.Request.Method == http.MethodGet {
			continue
		}

```

Reply:
won't work, as this info is not available from config.. it should be safe to remove this check